spacy/cli/project/dvc.py

```diff
@@ -105,6 +108,14 @@ def update_dvc_config(
         dvc_config_path.unlink()
     dvc_commands = []
     config_commands = {cmd["name"]: cmd for cmd in config.get("commands", [])}
+
+    # some flags that apply to every command
```

Comment:
Is there a reason why you removed the check for when both `quiet` and `verbose` were set? I felt it made sense to point out that contradiction when it happened? One thing we could do is make it a warning instead of a failure and opt for quiet as default?

Reply:
That was an oversight, thanks for catching it! However, it did make me wonder what happens if we pass both of these to DVC. It turns out it will fail with an error anyway:

```
ERROR: argument -q/--quiet: not allowed with argument -v/--verbose
```

So maybe there's no reason for us to catch this? I don't have strong feelings either way.

Reply:
Do those flags affect anything else than the DVC output?

Reply:
No, they're just passed through to DVC.

Reply:
My vote is for passing it through then and not adding another check. I'd let contradictory arguments fail with DVC.

Reply:
I'll just leave this check out then, so this should be ready to go.